kubejs/server_scripts/enigmatica/kubejs/functions.js

```diff
@@ -46,14 +46,14 @@ function wrapArray(array) {
     return utils.listOf(array).toArray();
 }
 
-function getStrippedLog(value) {
-    let strippedLog = air;
-    for (var i = 0; i < buildWoodVariants.length; i++) {
-        if (buildWoodVariants[i].logBlock === value) {
-            strippedLog = buildWoodVariants[i].logBlockStripped;
+function getStrippedLogFrom(logBlock) {
+    let result = air;
+    buildWoodVariants.find((wood) => {
+        if (wood.logBlock == logBlock) {
+            result = wood.logBlockStripped;
```

Comment:
The find function will continue searching after it has found it's result in this form, there needs to be a `break;` or `return result` here :)